.github/workflows/release.yaml

```diff
@@ -66,3 +66,25 @@ jobs:
           git add .
           git commit -m "chore(release): 📦 sync lockfile"
           git push origin changeset-release/main
+
+      - name: Send a discord notification
+        if: steps.changeset.outputs.hasChangesets == 'true'
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.MY_GITHUB_TOKEN }}
+          script: |
+            fetch(${{ secrets.DISCORD_WEBHOOK_URL }}, {
```

Comment:
Should look something like

https://discord.com/api/webhooks/<some numbers>/<a veeery long string>